modules/websearch/lib/search_engine_cvifier.py

```diff
@@ -53,24 +53,38 @@ def cvify_records(recids, of, req=None, so='d'):
     elif of == 'tlcv':
         HEADER = r'''
 \documentclass{article}
-\usepackage{textcomp} %% text version of symbols in TS1, e.g. \textrightarrow{}
-\usepackage{amstext} %% proper support of frequently used \text{} command
-%%To use pdflatex, uncomment these lines, as well as the \href lines
-%%in each entry
-%%\usepackage[pdftex,
-%%       colorlinks=true,
-%%       urlcolor=blue,       %% \href{...}{...} external (URL)
-%%       filecolor=green,     %% \href{...} local file
-%%       linkcolor=red,       %% \ref{...} and \pageref{...}
-%%       pdftitle={Papers by AUTHOR},
-%%       pdfauthor={Your Name},
-%%       pdfsubject={Just a test},
-%%       pdfkeywords={test testing testable},
-%%       pagebackref,
-%%       pdfpagemode=None,
-%%        bookmarksopen=true]{hyperref}
-%%usepackage{arial}
-%%\renewcommand{\familydefault}{\sfdefault} %% San serif
+\usepackage{textcomp} % text version of symbols in TS1, e.g. \textrightarrow{}
+\usepackage{amstext} % proper support of frequently used \text{} command
+\newif\ifshowcitations\showcitationsfalse
+\newif\ifshowlinks\showlinksfalse
+
+%% CONFIGURATION
+%
+%% to display citation counts, uncomment the following line
+%\showcitationstrue
+%
+%% to add links to INSPIRE records, uncomment the following line
+%\showlinkstrue
```

Comment:
I went the other direction and removed the pdftitle and pdfauthor keys from the hyperref config.
I think we should aim for simplicity here, they are not required and a knowledgeable user can always add them if she wants to.